Makefile

```diff
@@ -92,7 +92,6 @@ $(CILIUM_ENVOY_BIN) $(CILIUM_ENVOY_RELEASE_BIN): force
 	@$(ECHO_BAZEL)
```

Comment:
For this particular commit. Do we need to ship `cilium-envoy-unstripped` in the envoy docker image? This will increase the image size (by how much?), is it really necessary? why can't we create 2 docker images, one for debug and the other one containing the stripped binary only? I can help you with this if you want. I assume we only need `cilium-envoy-unstripped` for debug purposes so we can request for the backtrace and decode envoy stack traces locally.

Reply:
@aanm Help would be appreciated. To be sure the unstripped version is exactly the same build, we must only build once. But then we need to modify the Git repo for the "stripped build" to refer to the unstripped build, by the time which the SHA of the repo has now changed, and is not reflecting the build SHA anymore. I guess we could just tag the stripped image with the actual build SHA instead of the GIT SHA, but I'm open to other suggestions as well!

Reply:
@jrajahalme my suggestion will be to create a new repo, `quay.io/cilium/cilium-envoy-binaries`, that repository will need a new Dockerfile that creates envoy stripped and unstripped. 

Then, the ./Dockerfile on the root of this project would be something like:

```dockerfile
FROM quay.io/cilium/cilium-envoy-binaries:<commit-ID> as builder

FROM scratch
LABEL maintainer="maintainer@cilium.io"
COPY --from=builder /tmp/install/cilium-envoy-stripped /cilium-envoy
```
This one would be to create the image `quay.io/cilium/cilium-envoy`

And probably would be better to have another Dockerfile for the unstripped version:
```dockerfile
FROM quay.io/cilium/cilium-envoy-binaries:<commit-ID> as builder

FROM scratch
LABEL maintainer="maintainer@cilium.io"
COPY --from=builder /tmp/install/cilium-envoy-unstripped /cilium-envoy
```

This one would be to create the image `quay.io/cilium/cilium-envoy-unstripped`